tests/integration_tests/ui_tests/ArelleGUITest/ArelleGUITest/Tests.cs

```diff
@@ -0,0 +1,362 @@
+using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Definitions;
+using FlaUI.Core.Input;
+using FlaUI.Core.Logging;
+using FlaUI.Core.Tools;
+using FlaUI.Core.WindowsAPI;
+using FlaUI.TestUtilities;
+using FlaUI.UIA3;
+using System.Diagnostics;
+using System.Drawing;
+using System.IO.Compression;
+using System.Text;
+
+namespace ArelleGUITest
+{
+    [TestFixture]
+    public class ArelleGUITests : FlaUITestBase
+    {
+        protected override ApplicationStartMode ApplicationStartMode => ApplicationStartMode.OncePerTest;
+        protected override VideoRecordingMode VideoRecordingMode => VideoRecordingMode.OnePerTest;
+
+        abstract record ExecutionTypeT(string? ArellePath, string ArelleResourcesPath)
+        {
+            public record Source(string? ArellePath, string ArelleResourcesPath, bool UseVirtualEnv, string PythonExe)
+                : ExecutionTypeT(ArellePath, ArelleResourcesPath);
+            public record Build(string? ArellePath, string ArelleResourcesPath) : ExecutionTypeT(ArellePath, ArelleResourcesPath);
+        }
+
+        readonly ExecutionTypeT ExecutionType;
+        Window Window;
+
+        public ArelleGUITests()
+        {
+            var useSource = Environment.GetEnvironmentVariable("ARELLE_USE_BUILD") != "true";
+            // ARELLE_PATH
+            // source - path to checkout
+            // portable build - path to extract directory
+            // installed build - absent/empty, checks Program Files
+            // convert empty string to null to simplify GitHub Actions conditionals
+            var arellePathEnv1 = Environment.GetEnvironmentVariable("ARELLE_PATH");

```

Comment:
Not sure I understand why is this returning empty instead of `null`?

Reply:
No. I would expect it to return `null` if unset. Why does it return `''` sometimes?

Reply:
According to the [docs](https://learn.microsoft.com/en-us/dotnet/api/system.environment.getenvironmentvariable?view=net-7.0#system-environment-getenvironmentvariable(system-string)) its either been set and it returns the value or it hasn't been set and it returns `null`. I am not following why is it variably returning `null` sometimes and `''` other times, it should be one or the other consistently I would think